wordle-app/app.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <NuxtLayout>
+    <v-app :style="gradientBaseStyle" class="full-page">
+      <v-app-bar color="primary" :elevation="2">
+        <v-app-bar-title @click="$router.push('/')" style="cursor: pointer">
+          <v-img src="./public/image-Background-Removed.png" alt="Logo" max-width="180" max-height="95"></v-img>
+        </v-app-bar-title>
+
+        <v-btn v-if="!isLogged" @click="showLoginDialog = true" variant="tonal">
+          <v-icon class="mr-2">mdi-login</v-icon>
+          Log in
+        </v-btn>
+        <v-btn v-else @click="showLoginDialog = true" variant="tonal">
+          <v-icon class="mr-2">mdi-account</v-icon>
+          {{ tokenService.getUserName() }}
```

Comment:
love how you can see the username in the app bar once you login! Good design there

Reply:
Thank you :)